spec/requests/products/edit/edit_spec.rb

```diff
@@ -888,6 +888,43 @@
     expect(product.description).to eq("<p>Hi there!</p><review-card reviewid=\"#{review2.external_id}\"></review-card><review-card reviewid=\"#{review1.external_id}\"></review-card>")
   end
 
+  shared_examples "notifying users about content updates" do |factory_name|
+    it "allows notifying users" do
+      product = create(factory_name, user: seller, name: "Sample product", price_cents: 1000)
+
+      allow_any_instance_of(Link).to receive(:successful_sales_count).and_return(1)
```

Comment:
Can we create an actual purchase record so that this is test more resistant against implementation detail changes?